### DISCUSSION TITLE
After  diagrams.net upgrading to 21.1.2,  flowcharing(a grafana plugin) doesn't work.

### DISCUSSION THREAD

**Discussion by Kenny-SZ** [jgraph/drawio#3487] [Q&A]
After  diagrams.net upgrading to 21.1.2,  flowcharing(a grafana plugin) doesn't work.

The version of flowcharing is 0.9.1, before diagrams.net's upgradation, it worked correctly, even through it has the warning "app.diagrams.net will stop working for embed mode, please use embed.diagrams.net". 
It seems to do with "switch off compression" or "Compress, Encode", "Extract/Decode"?

When I edit diagrams on line， the url is “https://app.diagrams.net/?embed=1&spin=1&libraries=1&ui=dark&src=grafana”。

What  should I do to let it works? Thanks a lot in advance.

**Comment by davidjgraph** [comment]
It sounds like the grafana plugin doesn't support uncompressed diagrams. I would contact the author(s) of the plugin to see if they have an update to either support uncompressed data or force compressed mode in using the 21.1.4 release.

**Reply by Kenny-SZ** [reply]
Thanks, davidjgraph. 
If there is any update, please let me know.

**Reply by davidjgraph** [reply]
Update on what?

**Reply by Kenny-SZ** [reply]
> Update on what?

diagrams.net won't change its behavior, it needs the plugin to do some change to adjust to diagrams.net? Is my understanding correct?

**Reply by davidjgraph** [reply]
It needs an update to the grafana plugin. That's not an update from us.
